src/set_theory/ordinal.lean

```diff
@@ -1712,9 +1715,8 @@ open ordinal
 def ord (c : cardinal) : ordinal :=
 begin
   let ι := λ α, {r // is_well_order α r},
-  have : ∀ α, nonempty (ι α) := λ α,
-    ⟨classical.indefinite_description _ well_ordering_thm⟩,
-  let F := λ α, ordinal.min (this _) (λ i:ι α, ⟦⟨α, i.1, i.2⟩⟧),
+  have : Π α, ι α := λ α, ⟨well_ordering_rel, by apply_instance⟩,
```

Comment:
Given that this comes up in all three uses, perhaps define something of type `{r // is_well_order α r}`?